src/azul/service/elasticsearch_service.py

```diff
@@ -163,16 +175,22 @@ def _create_query(self, catalog: CatalogName, filters):
         for facet, values in filters.items():
             relation, value = one(values.items())
             if relation == 'is':
-                query = Q('terms', **{facet + '.keyword': value})
                 field_type = self.field_type(catalog, tuple(facet.split('.')))
-                translated_none = field_type.to_index(None)
-                if translated_none in value:
-                    # Note that at this point None values in filters have already
-                    # been translated eg. {'is': ['~null']} and if the filter has a
-                    # None our query needs to find fields with None values as well
-                    # as absent fields
-                    absent_query = Q('bool', must_not=[Q('exists', field=facet)])
-                    query = Q('bool', should=[query, absent_query])
+                if isinstance(field_type, Nested):
```

Comment:
Consider chaining on https://github.com/DataBiosphere/azul/pull/3706 It's gonna land first.

Reply:
PL this too, just want to know how it effects this PR.